circuits-circom/regexes/venmo_actor_id.circom

```diff
@@ -189,4 +189,21 @@ template VenmoActorId (msg_bytes) {
     for (var i = 0; i < num_bytes; i++) {
         reveal[i] <== in[i] * states[i+1][1];
     }
+
+    signal output reveal_start_idx;
+    signal prefix_sum[num_bytes+1];
+    signal temp_index[num_bytes+1];
+    component is_eq_one[num_bytes];
+    prefix_sum[0] <== 0;
+    temp_index[0] <== 0;
+    for (var i = 1; i <= num_bytes; i++) {
+        prefix_sum[i] <== prefix_sum[i-1] + (1 - states[i][1]);
```

Comment:
Should `1 - states[i][1]` be changed to `1 * states[i][1]` ? 

Also I think we can get rid of temp_index[num_bytes+1] and instead just do `reveal_start_idx <== prefix_sum[num_bytes]` right? 

I'm thinking if states[...][1] = 000011100 for example, then prefix_sum[...] = 000012333. So we just want to return the last value of prefix sum. Let me know if my thinking is correct.